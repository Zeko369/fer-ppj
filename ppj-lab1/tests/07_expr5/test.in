IDN 1 a
OP_PRIDRUZI 1 =
BROJ 1 1
OP_PLUS 1 +
BROJ 1 2
OP_PUTA 1 *
BROJ 1 3
OP_DIJELI 1 /
BROJ 1 4
