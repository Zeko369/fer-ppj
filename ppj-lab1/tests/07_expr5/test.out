<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 a
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 1
      <T_lista>
       $
     <E_lista>
      OP_PLUS 1 +
      <E>
       <T>
        <P>
         BROJ 1 2
        <T_lista>
         OP_PUTA 1 *
         <T>
          <P>
           BROJ 1 3
          <T_lista>
           OP_DIJELI 1 /
           <T>
            <P>
             BROJ 1 4
            <T_lista>
             $
       <E_lista>
        $
  <lista_naredbi>
   $
